app/views/budget_categories/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="devise-forms">
+  <h1>New budget category</h1>
+
+  <%= render "form", budget_category: @budget_category %>
+
+  <br>
+
+  <div>
+    <%= link_to "Back to budget categories", budget_categories_path %>
```

Comment:
- [ ] This back button should be **`<`** and should be on the top of the page based on the design.

Reply:
Done. Thanks